libuavcan/src/transport/uc_can_acceptance_filter_configurator.cpp

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (C) 2014 Pavel Kirienko <pavel.kirienko@gmail.com>,
+ *                    Ilia  Sheremet <illia.sheremet@gmail.com>
+ */
+
+#include <uavcan/transport/can_acceptance_filter_configurator.hpp>
+#include <cassert>
+
+namespace uavcan
+{
+const unsigned CanAcceptanceFilterConfigurator::DefaultFilterMsgMask;
+const unsigned CanAcceptanceFilterConfigurator::DefaultFilterServiceRequestID;
+const unsigned CanAcceptanceFilterConfigurator::DefaultFilterServiceRequestMask;
+const unsigned CanAcceptanceFilterConfigurator::ServiceRespFrameID;
+const unsigned CanAcceptanceFilterConfigurator::ServiceRespFrameMask;
+
+int16_t CanAcceptanceFilterConfigurator::loadInputConfiguration()
+{
+    multiset_configs_.clear();
+
+    CanFilterConfig service_resp_cfg;
+    service_resp_cfg.id = ServiceRespFrameID;
+    service_resp_cfg.mask = ServiceRespFrameMask;
+	if (multiset_configs_.emplace(service_resp_cfg) == NULL)
```

Comment:
Bad formatting (did you run uncrustify on this file?)

Also, constructor initializer lists are still formatted weirdly (the humanity would surely benefit if somebody developed an adequate autoformatter capable of handling initializer lists without breaking things); above I found this:

```c++
CanSelectMasks() :
    read(0), write(0)
{ }
```

Which is supposed to be:

```c++
CanSelectMasks() :
    read(0),
    write(0)
{ }
```

It's quite a minor and insignificant issue, I just want to get to the bottom of it: who formatted it this way - you or uncrustify? :) If it's uncrustify, what version did this? I thought I wrote a more or less adequate config that should instruct it to handle constructors properly.